### PR TITLE
Start authentication proxy when opening cluster settings

### LINKED ISSUE #1062: Cluster information fetch fails if opening cluster settings before cluster dashbord
**Describe the bug**
If opening cluster settings before opening the cluster dashboard for the same cluster, Lens is unable to fetch cluster information, because authentication proxy is not started.

### PR DESCRIPTION
**Pull request by nevalla**
Fixes #1062 

Signed-off-by: Lauri Nevala <lauri.nevala@gmail.com>

### CODE REVIEW

File: src/common/cluster-ipc.ts [hunk 1]
```diff
@@ -5,11 +5,10 @@ import { tracker } from "./tracker";
 export const clusterIpc = {
   activate: createIpcChannel({
     channel: "cluster:activate",
-    handle: (clusterId: ClusterId, frameId?: number) => {
+    handle: (clusterId: ClusterId, force = false) => {
```
Comment (nevalla) [inline src/common/cluster-ipc.ts:8]: we don't activate cluster with `frameId` anymore, so it's safe to remove it from here and replace with `force`

File: src/main/cluster.ts [hunk 2]
```diff
@@ -126,7 +127,10 @@ export class Cluster implements ClusterModel {
   }
 
   @action
-  async activate() {
+  async activate(force = false ) {
+    if (this.activated && !force) {
```
Comment (nevalla) [inline src/main/cluster.ts:131]: prevent activating cluster twice if activation has already been happened

File: src/renderer/components/+cluster-settings/cluster-settings.tsx [hunk 3]
```diff
@@ -46,8 +46,9 @@ export class ClusterSettings extends React.Component<Props> {
     }
   }
 
-  refreshCluster = () => {
+  refreshCluster = async () => {
     if(this.cluster) {
+      await clusterIpc.activate.invokeFromRenderer(this.cluster.id);
```
Comment (nevalla) [inline src/renderer/components/+cluster-settings/cluster-settings.tsx:51]: Activate cluster first if it's not yet activated

File: src/renderer/components/cluster-manager/cluster-status.tsx [hunk 4]
```diff
@@ -47,13 +47,14 @@ export class ClusterStatus extends React.Component<Props> {
...
-    await clusterIpc.activate.invokeFromRenderer(this.props.clusterId);
+  activateCluster = async (force = false) => {
+    await clusterIpc.activate.invokeFromRenderer(this.props.clusterId, force);
   }
 
   reconnect = async () => {
+    this.authOutput = []
     this.isReconnecting = true;
-    await this.activateCluster();
+    await this.activateCluster(true);
```
Comment (nevalla) [inline src/renderer/components/cluster-manager/cluster-status.tsx:57]: When reconnecting, we need to re-activate with force.

**Review by Nokel81 — APPROVED** [review]

**Pull request merged**